utils/zerovec/src/cow.rs

```diff
@@ -47,6 +48,35 @@ pub struct VarZeroCow<'a, V: ?Sized> {
 unsafe impl<'a, V: ?Sized> Send for VarZeroCow<'a, V> {}
 unsafe impl<'a, V: ?Sized> Sync for VarZeroCow<'a, V> {}
 
+impl<'a, V: ?Sized> Clone for VarZeroCow<'a, V> {
```

Comment:
Suggestion: maybe convert it to an `enum PrivateCow<'a, V> { Owned(Box<V>), Borrowed(&'a V) }`, where you can `#[derive(Clone)]`, and then you don't need to duplicate the unsafe code

Reply:
I'd still need unsafe code to extract the `Box<V>` and `&'a V`. These could be added as methods, but I think it only cleans up one method (this one).

Reply:
I still think the encapsulation would be better, but not strongly